runtime/tests/jvmtitests/module.xml

```diff
@@ -24,197 +24,213 @@
-		<export name="Java_com_ibm_jvmti_tests_verboseGC_vgc001_tryRegisterVerboseGCSubscriber3"/>
-		<export name="Java_com_ibm_jvmti_tests_verboseGC_vgc001_tryDeregisterVerboseGCSubscriber3"/>
-		<export name="Java_com_ibm_jvmti_tests_verboseGC_vgc001_getBufferCount3"/>
-		<export name="Java_com_ibm_jvmti_tests_verboseGC_vgc001_hasAlarmed3"/>
-		<export name="Java_com_ibm_jvmti_tests_getJ9vmThread_gjvmt001_validateGJVMT001"/>
-		<export name="Java_com_ibm_jvmti_tests_getJ9method_gj9m001_validateGJ9M001" />
-		<export name="Java_com_ibm_jvmti_tests_retransformRedefineCombo_rrc001_setTransformerData"/>
-		<export name="Java_com_ibm_jvmti_tests_retransformRedefineCombo_rrc001_retransformClass"/>
-		<export name="Java_com_ibm_jvmti_tests_retransformRedefineCombo_rrc001_redefineClass"/>
-		<export name="Java_com_ibm_jvmti_tests_redefineBreakpointCombo_rbc001_redefineClass"/>
-		<export name="Java_com_ibm_jvmti_tests_redefineBreakpointCombo_rbc001_setBreakpoint"/>
-		<export name="Java_com_ibm_jvmti_tests_redefineBreakpointCombo_rbc001_clearBreakpoint"/>
-		<export name="Java_com_ibm_jvmti_tests_redefineBreakpointCombo_rbc001_setBreakpointInMethodID"/>
-		<export name="Java_com_ibm_jvmti_tests_redefineBreakpointCombo_rbc001_getMethodID"/>
-		<export name="Java_com_ibm_jvmti_tests_redefineBreakpointCombo_rbc001_getMethodIDFromStack"/>
-		<export name="Java_com_ibm_jvmti_tests_classModificationAgent_cma001_retransformClass"/>
-		<export name="Java_com_ibm_jvmti_tests_classModificationAgent_cma001_redefineClass"/>
-		<export name="Java_com_ibm_jvmti_tests_registerNativesWithRetransformation_rnwr001_registerNative" />
-		<export name="Java_com_ibm_jvmti_tests_registerNativesWithRetransformation_rnwr001_retransformClass" />
-		<export name="Java_com_ibm_jvmti_tests_agentLibraryNatives_aln001_shortname" />
-		<export name="Java_com_ibm_jvmti_tests_agentLibraryNatives_aln001_longname__I" />
-		<export name="Java_com_ibm_jvmti_tests_modularityTests_mt001_addModuleReads" />
-		<export name="Java_com_ibm_jvmti_tests_modularityTests_mt001_addModuleExports" />
-		<export name="Java_com_ibm_jvmti_tests_modularityTests_mt001_addModuleUses" />
-		<export name="Java_com_ibm_jvmti_tests_modularityTests_mt001_addModuleOpens" />
-		<export name="Java_com_ibm_jvmti_tests_modularityTests_mt001_addModuleProvides" />
-		<export name="Java_com_ibm_jvmti_tests_nestMatesRedefinition_nmr001_redefineClass" />
-		<export name="Java_com_ibm_jvmti_tests_setNativeMethodPrefix_snmp001_setPrefix" />
-		<export name="Java_com_ibm_jvmti_tests_setNativeMethodPrefix_snmp001_nat" />
-		<export name="Java_com_ibm_jvmti_tests_samplingObjectAllocation_soae001_reset" />
-		<export name="Java_com_ibm_jvmti_tests_samplingObjectAllocation_soae001_enable" />
-		<export name="Java_com_ibm_jvmti_tests_samplingObjectAllocation_soae001_disable" />
-		<export name="Java_com_ibm_jvmti_tests_samplingObjectAllocation_soae001_check" />
-	</exports>
-
+	<exportlists>
```

Comment:
Nesting within `<exportlists>` isn't necessary; please remove to minimize changes required.